src/app/pages/edit-profile/edit-profile.controller.js

```diff
@@ -18,26 +18,25 @@ function EditProfileController($log, $scope, $rootScope, userProfileService, syn
       if (file) {
         userProfileService.setProfileImage(file);
       }
+
       $scope.success = true;
-      setTimeout(function () {
+      $timeout(function () {
         $state.go('profile');
       }, 1500);
```

Comment:
why do you decide use this time for waiting?

Reply:
I based on my own experience, I tried to use another time, but this time seemed to me optimal.

Reply:
and this time will be enought for reading message about successful saving.

Reply:
I delete this $timeout.